packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/checkfortoken.md

```diff
@@ -1,36 +1,42 @@
-Check for an existing session in two steps.
+To check for an existing session may take two steps:
```

Comment:
If these steps are required, are they both required? If not, it's probably better to use bullet points.

Reply:
Nice catch! The numbered steps were from the older version which required both steps. I've updated them to bullets. That will appear on the next checkin.